gravity-forms/gw-cache-buster.php

```diff
@@ -205,6 +209,14 @@ public function suppress_default_post_render_event( $form_string, $form, $curren
 			$form_string = str_replace( $search, '', $form_string );
 		}
 
+		if ( is_callable( 'GFFormDisplay::post_render_script' ) ) {
+			$post_render_script = GFFormDisplay::post_render_script( $form['id'], $current_page );
+			// $post_render_script    = GFCommon::get_inline_script_tag( $post_render_script );
+			$post_render_script = preg_quote( $post_render_script, '/' ); // Escape special characters
+			$pattern            = '/gform\.initializeOnLoaded\(\s*function\(\)\s*\{\s*(' . $post_render_script . ')\s*\}\s*\);/';
+			$form_string        = preg_replace( $pattern, '', $form_string );
```

Comment:
This does leave an empty `<script></script>` in `$form_string`. I will address that when we confirm that this fix is going to work but I don't want to waste time on that if the approach needs changed.

Reply:
This works for me!

Reply:
Resolved in 824c17801a6b0736efbb121d2ba634dc09fb2610